creator-node/src/logging.js

```diff
@@ -3,13 +3,35 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+// taken from: https://github.com/trentm/node-bunyan/issues/194#issuecomment-347801909
+// since there is no official support for string-based "level" values
+// response from author: https://github.com/trentm/node-bunyan/issues/194#issuecomment-70397668
+function RawStdOutWithLevelName() {
+  return {
+    write: (log) => {
+      // duplicate log object before sending to stdout
+      const clonedLog = Object.assign({}, log)
+
+      // rename level (int) to levelno key
+      clonedLog.levelno = clonedLog.level
```

Comment:
btw i didn't rename this on purpose. this messes up the base bunyan formatting when it auto prints out level. instead of INFO it's LVLinfo eg. that's why i called the string logLevel
```
[2022-05-23T23:56:53.943Z]  INFO: audius_creator_node/7044 on 39be7272cc00: getNodeUsers() nodeUsers.length: 0
 it's
[2022-05-23T23:37:59.123Z] LVLinfo: audius_creator_node/6026 on 39be7272cc00: Loglevel set to: debug (levelno=30)
```

Reply:
Sounds good. I made the matching change in [DP](https://github.com/AudiusProject/audius-protocol/pull/3124/files#diff-66a46cd6374fa0e7843ebcad3becef15ca0cfb4822acb3a5e964490ab8be18fc) as well.